content/knowledge/flare_architecure.md

```diff
@@ -0,0 +1,192 @@
+It is a real-world representation of assets such as bonds and equity with specific rules that govern their actions.
+Whenever you initiate a transaction on the Avalanche blockchain, you get to pay a fee in AVAX.
+The Exchange Chain is an Avalanche Virtual Machine (AVM) example.
+Clients can create and trade assets on the X-Chain and other examples of the AVM with the help of the X-Chain API.
+
+## Flare's FBA
+
+_TODO_ add more details to each of the items
+
+Flare uses some parts of the Avalanche codebase with some changes:
+
+* deactivate most of the P-Chain and X-Chain functionality;
+* run the C-Chain with a locally defined set of validators instead of using the P-Chain for the list of validators;
+* hijack the EVM state transition to implement Flare state connector functionality;
+* implement the rest of the Flare systems on top of the C-Chain as Solidity smart contracts.
+
+## State Connector
+
+The state connector in Flare's system is designed to observe the state of underlying chains and is one of the most critical protocols on the Flare Network.
+The state connector enables Flare smart contracts to act on state changes from underlying blockchains, for example, payments.
+The state connector adopts other blockchains' consensus mechanisms and validator ecosystems.
+
+### Advantages
+
+The [state connector system](https://docs.flare.network/en/state-connector) is a competitive approach for proving the state of an underlying chain to a smart contract, and it has the following advantages:
+
+* Transaction validity references back to an underlying chain's genesis block: Other approaches like the SPV (Simplified Payment Verification) proof do not check the validity of a transaction because it is considered harmful for proving the state of an underlying chain to another network.
+* Safety only depends on an underlying chain's validators: no trusted third-party service has its own set of economic incentives and risks.
+  The need for trust is minimized by leveraging the guarantee that safety can only be lost in the state connector if an underlying chain's validators encounter a Byzantine fault.
+* No cooperation is needed from an underlying chain's validators: validators from an underlying chain do not need to modify their chain's codebase to permit Flare to interpret their network.
+  An underlying chain's validators do not even need to know that Flare exists for the state connector system to operate.
+* Can read the state of any blockchain: the state connector can operate on any possible Sybil-resistance technique of an underlying chain.
+* Constant-sized proofs: both the data availability proof and the payment proof are constant-sized, independent of the number of other payments in the data availability period being considered.
+
+### Voting
+
+There are three phases of the state connector voting protocol:
+
+* Request
+* Commit
+* Reveal
```

Comment:
```suggestion
The state connector voting protocol is split into three phases, described in the following sub-sections.
```

WDYT? I feel like this list is a bit redundant with the sections below, since it just lists the names of the sections.

Reply:
Fixed.